src/main/java/org/threadly/litesockets/utils/MergedByteBuffers.java

```diff
@@ -17,13 +17,13 @@
  */
 public class MergedByteBuffers {
```

Comment:
I think the main thing with ByteBuffers is they wont throw exceptions if using multiThreads.  In either case I agree it nees to be documented (I thought it was).